src/ui/views/ChainManagement/index.tsx

```diff
@@ -10,23 +11,21 @@ import './style.less';
 export const ChainManagementList = () => {
   const wallet = useWallet();
   const { t } = useTranslation();
-  const [allChains, setAllChains] = useState<Chain[]>([]);
-  const [chains, setChains] = useState<Chain[]>([]);
+  const [allChains, setAllChains] = useState<Chain[]>(Object.values(CHAINS));
```

Comment:
A constant no need to set as state right?

Reply:
done